source/write/gridfs.txt

```diff
@@ -46,7 +46,7 @@ defined in the GridFS specification:
 - The ``files`` collection stores the file metadata.
 
 When you create a new GridFS bucket, the driver creates the ``fs.chunks`` and ``fs.files``
-collections, unless you specify a different name in the ``Grid::FSBucket.new`` method options. The
+collections, unless you specify a different name in the ``Mongo::Database.fs`` method options. The
```

Comment:
Minor thing: when referring to an instance method of a class, use a `#` character instead of the period. The period suggests a class method, instead of an instance method.

```suggestion
collections, unless you specify a different name in the ``Mongo::Database#fs`` method options. The
```